tools/cppcheck/suppressions-list

```diff
@@ -1,3 +1,4 @@
 operatorEqVarError
 noConstructor
 duplicateExpression
+variableScope
```

Comment:
That's the ID cppcheck reported for the error when running with --xml. I couldn't find any detailed description of the error, nor how many checks this suppression would turn off.

In function `ecma_builtin_helper_object_to_string` the following is reported:
```
[jerry-core/ecma/builtin-objects/ecma-builtin-helpers.cpp:87]: (style) The scope of the variable 'buffer_size' can be reduced.
```
But I couldn't find any way to get rid of the error (moved the declaration earlier, moved a var declaration between the declaration of `buffer_size` az the local array allocation, removed the const specifier, etc.). Anyway, they would only have been hacks. Working around false positives of a static checker cannot be the motivation for refactoring the code so I dropped that idea and suppressed the error ID.

Not sure about alternatives.